.idea/codeStyles/Project.xml

```diff
@@ -1,42 +1,10 @@
 <component name="ProjectCodeStyleConfiguration">
```

Comment:
Are these changes intentional?

Reply:
Good catch. Reverted in 3d43b3c2ca0d96045f697c1072cee483956de918